README.md

```diff
@@ -145,17 +146,38 @@ docker run -p 7687:7687 -p 7444:7444 memgraph/memgraph-mage
 
 #### 2 Install MAGE with Docker build of the repository
 
-**0.** Make sure that you have cloned the MAGE Github repository and positioned
+**0. a** Make sure that you have cloned the MAGE Github repository and positioned
 yourself inside the repo in your terminal:
 
 ```bash
 git clone --recurse-submodules https://github.com/memgraph/mage.git && cd mage
 ```
 
-**1.** To build the **MAGE** image run the following command:
+**0. b** Download memgraph from our official download site inside memgraph mage. Change ${MEMGRAPH_VERSION} with latest version Memgraph released, and
```

Comment:
```suggestion
**0. b** Download Memgraph from our official download site inside Memgraph MAGE. Change ${MEMGRAPH_VERSION} with latest version Memgraph released, and
```

Reply:
I don't find the first sentence very clear, if my understanding is correct would suggest
```suggestion
**0. b** Download Memgraph from our official site inside your cloned MAGE repository. Change ${MEMGRAPH_VERSION} with latest version Memgraph released, and
```